### PR TITLE
Create dianas-suggestion-day4.tweet

### PR DESCRIPTION
**Pull request by dianaiusan**

### CODE REVIEW

**Comment by bast** [comment]
Thanks for the PR! Only suggestion for later is to use actual 2022/03/... instead of the YYYY/MM path. This makes it easier to reuse these later and to find them.

**Comment by dianaiusan** [comment]
The very first commit was done on GitHub. I thought it would end in the correct place by default. It seems I was wrong. Should I redo the PR? I am OK with it. At least it will be easier next time.

**Comment by dianaiusan** [comment]
Hold on. I am going to fix this as it is bothering me.

**Comment by bast** [comment]
But it will also work like this. So we don't have to fix the path.

File: tweets/2022/03/dianas-suggestion-day4.tweet [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Excited for the start of the second week of the CodeRefinery (@coderefine) online #workshop!
```
Comment (bast) [inline tweets/2022/03/dianas-suggestion-day4.tweet:1]:
```suggestion
Excited for the start of the second week of the CodeRefinery (@coderefine) online workshop!
```

**Review by bast — APPROVED** [review]

**Pull request merged**